code/modules/mob/living/silicon/pai/pai_defense.dm

```diff
@@ -41,34 +40,36 @@
 			Paralyze(200)
 
 /mob/living/silicon/pai/attack_hand(mob/living/carbon/human/user, list/modifiers)
-	if(user.combat_mode)
-		user.do_attack_animation(src)
-		if (user.name == master)
-			visible_message(span_notice("Responding to its master's touch, [src] disengages its holochassis emitter, rapidly losing coherence."))
-			if(do_after(user, 1 SECONDS, TRUE, src))
-				fold_in()
-				if(user.put_in_hands(card))
-					user.visible_message(span_notice("[user] promptly scoops up [user.p_their()] pAI's card."))
-		else
-			visible_message(span_danger("[user] stomps on [src]!."))
-			take_holo_damage(2)
-	else
+	if(!user.combat_mode)
 		visible_message(span_notice("[user] gently pats [src] on the head, eliciting an off-putting buzzing from its holographic field."))
+		return
+	user.do_attack_animation(src)
+	if(user.name != master)
+		visible_message(span_danger("[user] stomps on [src]!."))
+		take_holo_damage(2)
+		return
+	visible_message(span_notice("Responding to its master's touch, [src] disengages its holochassis emitter, rapidly losing coherence."))
+	if(!do_after(user, 1 SECONDS, TRUE, src))
+		return
+	fold_in()
+	if(user.put_in_hands(card))
+		user.visible_message(span_notice("[user] promptly scoops up [user.p_their()] pAI's card."))
 
 /mob/living/silicon/pai/bullet_act(obj/projectile/Proj)
 	if(Proj.stun)
 		fold_in(force = TRUE)
 		src.visible_message(span_warning("The electrically-charged projectile disrupts [src]'s holomatrix, forcing [src] to fold in!"))
 	. = ..(Proj)
 
-/mob/living/silicon/pai/IgniteMob(mob/living/silicon/pai/P)
-	return FALSE //No we're not flammable
+/mob/living/silicon/pai/IgniteMob()
+	fire_stacks = 0
+	. = ..()
```

Comment:
Does this work?

Reply:
I actually just copy pasted this from AI

Reply:
Ah yes it does